package.json

```diff
@@ -84,7 +102,12 @@
     ],
     "moduleNameMapper": {
       "^react-native$": "react-native-web"
-    }
+    },
+    "modulePaths": [
```

Comment:
Are you sure this won't hurt more than help?
I really like to have full IDE support....